templates/next/app/frames/route.tsx

```diff
@@ -42,7 +43,7 @@ const handleRequest = frames(async (ctx) => {
     ),
     buttons: !ctx.url.searchParams.has("saidGm")
       ? [
-          <Button action="post" target={{ query: { saidGm: true } }}>
+          <Button action="post" key="1" target={{ query: { saidGm: true } }}>
```

Comment:
why is this key attribute required?

Reply:
eslint complains

Reply:
in React world each time you use array you should also use `key` even if this case it doesn't make any sense as we are not really using that. Will suppress that.

Reply:
yeah the user shouldn't have to worry about this

Reply:
removed